source/shared_lib/include/platform/sdl/platform_main.h

```diff
@@ -212,15 +212,18 @@ void printParameterHelp(const char *argv0, bool foundInvalidArgs) {
 	printf("\n\n                     \tWhere x is an optional name of the saved game file to load.");
 	printf("\n\n                     \tIf x is not specified we load the last game that was saved.");
 
-	printf("\n\n%s=x,y,z  \tRun in auto test mode.", GAME_ARGS[GAME_ARG_AUTO_TEST]);
-	printf("\n\n                     \tWhere x is an optional maximum # seconds to play.");
-	printf("\n\n                     \tIf x is not specified the default is 1200 seconds (20 minutes).");
-	printf("\n\n                     \tWhere y is an optional game settings file to play.");
-	printf("\n\n                     \tIf y is not specified (or is empty) then auto test cycles");
-	printf("\n\n                     \tthrough playing scenarios.");
-	printf("\n\n                     \tWhere z is the word 'exit' indicating the game should exit");
-	printf("\n\n                     \tafter the game is finished or the time runs out. If z is");
-	printf("\n\n                     \tnot specified (or is empty) then auto test continues to cycle.");
+	printf("\n\n\
+%s=x,y,z\n\
+  Run in auto test mode\n\n\
```

Comment:
I think I'd remove the 2nd newline here. I think indenting the description should be sufficient.